src/coreclr/vm/dllimport.h

```diff
@@ -270,20 +281,14 @@ inline bool SF_IsSharedStub(DWORD dwStubFlags)
         return false;
     }
 
+    if (SF_IsForwardPInvokeStub(dwStubFlags))
```

Comment:
Should this exclude Delegate and Calli like in the other place?

Reply:
Yep. I just got back from a run and am in the middle of cleaning that up. Occurred to me while running :)